code/game/objects/random/guns.dm

```diff
@@ -166,7 +166,9 @@
 				/obj/item/gun/projectile/boltgun/handmade = 1,\
 				/obj/item/gun/projectile/shotgun/slidebarrel = 1,\
 				/obj/item/gun/energy/laser/makeshift = 1,\
-				/obj/item/gun/energy/lasersmg = 1))
+				/obj/item/gun/energy/lasersmg = 1,\
+				/obj/item/gun/projectile/automatic/ak47/makeshift = 1,\
```

Comment:
Outsider buff for the win!

Reply:
Rebel said it was an oversight so I added it and honestly it's an early game nerf if you get the kalash as you need to waste points off your ammo kit for a mag, meanwhile with the bolt action you can just get an entire box and manually load it